lib/matplotlib/axes/_axes.py

```diff
@@ -8136,6 +8136,13 @@ def specgram(self, x, NFFT=None, Fs=None, Fc=None, detrend=None,
         data : indexable object, optional
             DATA_PARAMETER_PLACEHOLDER
 
+        vmin, vmax : float, optional
+            vmin and vmax define the data range that the colormap covers.
+            By default, the colormap covers the complete value range of the
+            data. It is an error to use vmin/vmax when a norm instance is
+            given (but using a str norm name together with vmin/vmax is
+            acceptable). This parameter is ignored if X is RGB(A).
```

Comment:
```suggestion
            data.
```
These parts are not relevant for `specgram`. You can't pass a norm or RGB(A) data.